requirements/tests.txt

```diff
@@ -6,5 +6,7 @@ cov-core==1.15.0
 sphinx==2.2.1
 sphinx-rtd-theme==0.4.3
 mock_msm~=0.9.2
+
+ovos-stt-plugin-selene>=0.0.3a3
```

Comment:
Same as above, this should have a stable version spec but could be handled in a separate PR